pages/dkp/kommander/1.3/projects/project-deployments/index.md

```diff
@@ -0,0 +1,99 @@
+excerpt: Project Deployments can be used to manage GitOps based Continuous Deployments which can automatically be created on all the Kubernetes clusters associated with the Project, in the corresponding namespace.
+---
+
+Kommander Projects can be configured with GitOps based Continuous Deployments for federation of your Applications to associated clusters of the project. This is backed by Dispatch which enables software and applications to be continuously deployed (CD) using GitOps processes. GitOps enables the application to be deployed as per a manifest that is stored in a Git repository. This ensures that the application deployment can be automated, audited and declaratively deployed to the infrastructure.
+
+## What is GitOps?
+
+GitOps is a modern software deployment strategy. The configuration describing how your application is deployed to a cluster are stored in a Git repository. The configuration is continuously synchronized from this Git repository to the cluster, ensuring that the specified state of the cluster always matches what is defined in the "GitOps" Git repository.
+
+The benefits of following a GitOps deployment strategy are:
+
+* Familiar, collaborative change and review process. Engineers are intimately familiar with Git-based workflows: branches, pull requests, code reviews, etc. GitOps leverages this experience to control deployment of software and updates to catch issues early.
+* Clear change log and audit trail. The Git commit log serves as an audit trail to answer the question: "who changed what, and when?" Having such information readily available allows you to reach out to the right people when fixing or triaging a production incident to determine the why and correctly resolve the issue as quickly as possible. Additionally, Dispatch's CD component (Flux CD) maintains a separate audit trail in the form of Kubernetes Events, as changes to a Git repository don't include exactly when those changes were deployed.
+* Avoid configuration drift. The scope of manual changes made by operators expands over time. It soon becomes difficult to know which cluster configuration is critical and which is left over from temporary workarounds or live debugging. Over time, changing a project configuration or replicating a deployment to a new environment becomes a daunting task. GitOps supports simple, reproducible deployment to multiple different clusters by having a single source of truth for cluster and application configuration.
+
+That said, there are some cases when live debugging is necessary in order to resolve an incident in the minimum amount of time. In such cases, a pull request-based workflow adds expensive overhead when you need it least. Dispatch's CD strategy supports this scenario by letting you disable the Auto Sync feature. After Auto Sync is disabled, Dispatch will stop synchronizing the cluster state from the GitOps git repository. This lets you use kubectl, helm or whichever tool you need to resolve the issue.
```

Comment:
"a pull request-based workflow adds expensive overhead when you need it least" I would prefer we highlight the time to resolution instead of the complexity of the pull-request process. What is expensive in this situation is the loss of production uptime. Not the process itself. "a pull-request-based workflow adds precious time to time to resolution for critical production outages." You can always commit the changes made during the outage back to GitOps and restore the configuration drift caused by the outage.